js/repl/Utils.js

```diff
@@ -0,0 +1,42 @@
+const sizes = ["Bytes", "kB", "MB", "GB", "TB", "PB", "EB"];
+export function prettySize(
+  size: number,
+  nospace: boolean,
+  one: boolean,
+  places: number
+) {
```

Comment:
Tiny nit, but it looks like this (forked) implementation is more complex than is needed for this project, since we only ever call it with a single (`size`) argument.

Can we drop things like `nospace`?